docs/development/contributing.md

```diff
@@ -105,9 +106,9 @@ be assigned by the Technical Board or Maintainers:
 | technical board | This label is given for PRs which will be discussed in a meeting of the Technical Board. The label will be removed after the discussion.                                  |
 
```

Comment:
The following Labels are missing: 
* dependencies
* documentation
* javascript
* php
* roadmap